src/stream.js

```diff
@@ -19,7 +19,7 @@ var http = require('./_http')
 var q = require('./query')
 var util = require('./_util')
 
-var DefaultEvents = ['start', 'error', 'version', 'history_rewrite']
+var DefaultEvents = ['start', 'error', 'version', 'history_rewrite', 'set']
```

Comment:
Later in this file, in lines 286-295, there is a doc block describing the event options, specifically the `fields` options. We need to include coverage of the `index` field.

Also, since client applications might use the available field names, should we define constants for the values? Seems like a miss to have developers use the literal `'index'` to get the `index` field in event objects.

Reply:
> Later in this file, in lines 286-295, there is a doc block describing the event options, specifically the fields options. We need to include coverage of the index field.

I'll update the JS doc.

> Also, since client applications might use the available field names, should we define constants for the values? Seems like a miss to have developers use the literal 'index' to get the index field in event objects.

I'm unfamiliar with JS library development best practices. I _think_ I'd rather focus on the minimum implementation to enable set-streaming by extending existing behaviour and/or ergonomics while leaving future improvements to next iterations.